<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Check</class>
 <widget class="QDialog" name="Dialog_Check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>check 2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_money_2">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_money_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_bank_2">
       <property name="text">
        <string>Bank</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_bank_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shobe_2">
       <property name="text">
        <string>Shobe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shobe_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_date_2">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shenase_2">
       <property name="text">
        <string>Shenase Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shenase_2"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>check 3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_money_3">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_money_3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_bank_3">
       <property name="text">
        <string>Bank</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_bank_3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shobe_3">
       <property name="text">
        <string>Shobe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shobe_3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_date_3">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shenase_3">
       <property name="text">
        <string>Shenase Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shenase_3"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>ch3ck 4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_money_4">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_money_4"/>
     </item>
     <item>
      <widget class="QLabel" name="label_bank_4">
       <property name="text">
        <string>Bank</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_bank_4"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shobe_4">
       <property name="text">
        <string>Shobe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shobe_4"/>
     </item>
     <item>
      <widget class="QLabel" name="label_date_4">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_4"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shenase_4">
       <property name="text">
        <string>Shenase Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shenase_4"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>ch3ck 5</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_money_5">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_money_5"/>
     </item>
     <item>
      <widget class="QLabel" name="label_bank_5">
       <property name="text">
        <string>Bank</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_bank_5"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shobe_5">
       <property name="text">
        <string>Shobe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shobe_5"/>
     </item>
     <item>
      <widget class="QLabel" name="label_date_5">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_5"/>
     </item>
     <item>
      <widget class="QLabel" name="label_shenase_5">
       <property name="text">
        <string>Shenase Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_shenase_5"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="check1">
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QLabel" name="label_money">
        <property name="text">
         <string>Money</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_money"/>
      </item>
      <item>
       <widget class="QLabel" name="label_bank">
        <property name="text">
         <string>Bank</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_bank"/>
      </item>
      <item>
       <widget class="QLabel" name="label_shobe">
        <property name="text">
         <string>Shobe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_shobe"/>
      </item>
      <item>
       <widget class="QLabel" name="label_date">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_shenase">
        <property name="text">
         <string>Shenase Check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_shenase"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog_Check</receiver>
   <slot>check_1_available(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog_Check</receiver>
   <slot>check_2_available(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog_Check</receiver>
   <slot>check_3_available(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog_Check</receiver>
   <slot>check_4_available(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_5</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog_Check</receiver>
   <slot>check_5_available(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check_1_available(bool)</slot>
  <slot>check_2_available(bool)</slot>
  <slot>check_3_available(bool)</slot>
  <slot>check_4_available(bool)</slot>
  <slot>check_5_available(bool)</slot>
 </slots>
</ui>
